Cargo.toml

```diff
@@ -49,7 +49,8 @@ native = [
 
 wasm = [
     "bevy_rapier3d/wasm-bindgen",
-    "core"
+    "core",
+    "wasm-bindgen",
```

Comment:
I think this syntax works here:
```suggestion
    "dep:wasm-bindgen",
```

Reply:
I realize I could use this syntax in some other optional deps as well, hehe

Reply:
Uu what is this syntax?